codec/codec-fury/src/main/resources/user.json

```diff
@@ -0,0 +1,3 @@
+{
+  "com.alipay.sofa.rpc.codec.fury.model.Registered.*": true
```

Comment:
Which file and format dubbo use for configuring whitelist/blacklist ?

Reply:
Thank you for your help, I will refer to it here